code/game/objects/items/devices/taperecorder.dm

```diff
@@ -403,6 +417,19 @@
 	cut_overlay("ribbonoverlay")
 	unspooled = FALSE
 
+/obj/item/tape/proc/wipeproc()
+	used_capacity = 0;
+	storedinfo = new;
+	timestamp = new;
+
+/obj/item/tape/verb/wipeverb()
+	set name = "Wipe Tape";
+	if(unspooled)
+		to_chat(usr, "<span class='notice'>You scrub the magnetic strip clean of its contents.")
+		wipeproc()
+	else if(!unspooled)
```

Comment:
```suggestion
	else
```
The `if` part of this statement is redundant

Reply:
Can you explain this one? I thought Byond couldn't handle 'else' alone.